libpod/events.go

```diff
@@ -105,9 +105,11 @@ func (c *Container) newContainerExitedEvent(exitCode int32) {
 	intExitCode := int(exitCode)
 	e.ContainerExitCode = &intExitCode
 
-	e.Details = events.Details{
-		Attributes: c.Labels(),
+	attrs := c.Labels()
+	if c.state.OOMKilled {
+		attrs["OOMKilled"] = "true"
```

Comment:
This doesn't really feel correct. I'd add a new field to the Event structure like `ContainerExitCode` - maybe call it OOMKilled - and set it there, as opposed to sticking it into the labels. Labels should be reserved for user-specified labels.